prow/proxy-postsubmit.sh

```diff
@@ -45,3 +45,4 @@ export BAZEL_BUILD_ARGS="--local_ram_resources=12288 --local_cpu_resources=8 --v
 echo 'Create and push artifacts'
 make push_release RELEASE_GCS_PATH="gs://istio-build/proxy"
 make artifacts ARTIFACTS_GCS_PATH="gs://istio-artifacts/proxy/${GIT_SHA}/artifacts/debs"
+./scripts/release-docker.sh
```

Comment:
Could you pull this back into `release-binary.sh`? Building it separately is going to trash Bazel's cache.

Also, GKE uses "make push_release" target to publish binaries, and since we expose all well-defined targets in the `Makefile`, scripts shouldn't be accessed directly. This alone could be fixed by adding this to `Makefile`, but cache trashing is the bigger concern...

Reply:
Refactored the script since it's becoming unmaintainable. Please take a look at it now.